swagger.yml

```
@@ -3835,6 +3862,29 @@ definitions:
             type: string
           secret_key:
             type: string
+
+  tenantPod:
+    type: object
+    required:
+      - name
+    properties:
+      name:
+        type:
+          string
+      status:
+        type:
+          string
+      timecreated:
```

Please use camel case:
```suggestion
      timeCreated:
```